index.html

```diff
@@ -0,0 +1,204 @@
+<!doctype html>
+	<html>
+		<head>
+			<meta charset="utf-8">
+			<meta http-equiv="X-UA-Compatible" content="IE=edge">
+			<title></title>
+			<meta name="description" content="">
+			<meta name="viewport" content="width=device-width, initial-scale=1">
+			<link href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css" rel="stylesheet">
+			<link rel="stylesheet" type="text/css" href="styles.css">
+		</head>
+		<body>
+			<nav class='navbar navbar-fixed-top'>
+				<div class="container">
+					<div class="navbar-header">
+						<a href="#">
+							<h1>Danebook</h1>
+						</a>
+					</div>
+					<div class="navbar-collapse collapse">
+						<form class="navbar-form navbar-right">
+							<div class="form-group">
+								<input class="form-control" type='text' placeholder='Email'></input>
+							</div>
+							<div class="form-group">
+								<input class="form-control" placeholder='password' type='password'></input>
+							</div>
+							<button class="btn btn-primary bordered">Log In</button>
+						</form>
+					</div>
+				</div>
+			</nav>
+
+			<div id='homepage-left' class="container">
+				<div class="nav-spacer"></div>
```

Comment:
You can also add some space with </br> tags (which is what I did), but your solution better because it's simple edit to the CSS file to adjust the width.